wg-api-expression/README.md

```diff
@@ -0,0 +1,32 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# API Expression Working Group
+
+Enable API authors to better serve API consumers, by improving and documenting all aspects of Kubernetes API development.
+[See full Mission Statement](https://docs.google.com/document/d/1XYbQXfge2qKM9psksfC5XZnW8hybtLqL1EcJLU4JwKg).
+
+## Stakeholder SIGs
+* SIG API Machinery
+* SIG Architecture
+
+## Meetings
+* Regular WG Meeting: [Tuesdays at 9:30 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
+
+## Organizers
+
+* Antoine Pelisse (**[@apelisse](https://github.com/apelisse)**), Google
+* Kevin Wiesmueller (**[@kwiesmueller](https://github.com/kwiesmueller)**), //SEIBERT/MEDIA GmbH
+
+## Contact
+- Slack: [#wg-api-expression](https://kubernetes.slack.com/messages/wg-api-expression)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-api-expression)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fapi-expression)
+<!-- BEGIN CUSTOM CONTENT -->
+
```

Comment:
Can we put the full mission statement here?

Reply:
Only if we change the generation Tool to use a different yaml style for the missionStatement field.

I can file a PR and see what people say.

Reply:
Oh wait, you mean at the custom content tag.
Didn't see that, I'll try to find out more.

Reply:
Are you sure you can't just directly edit this .md file here? The `<!-- BEGIN CUSTOM CONTENT -->` comment strongly implies that you can.

Reply:
You're right.
Updated.